packages/spruce-skill-server/gql/router.ts

```diff
@@ -100,12 +98,16 @@ export default (
 				formatError: (e: Error) => {
 					const code = e.message
 					let formattedError: Record<string, any> = {}
-
+					const errors = config.errors
+					// @ts-ignore
 					if (errors[code]) {
 						formattedError = {
 							name: code,
+							// @ts-ignore
 							code: errors[code].code,
```

Comment:
For situations like this, is it practical to put a `get` method on both errors and config like this:

```
config.get(key: [K as keyof config]): config[k]
```
I can't remember the exact syntax, but i did something like this before where it only let you pass keys and it wokred pretty well.